apm-agent-plugins/apm-spring-webflux/apm-spring-webflux-plugin/src/test/java/co/elastic/apm/agent/springwebflux/AbstractServerInstrumentationTest.java

```diff
@@ -72,6 +72,7 @@ static void stopApp() {
     void beforeEach() {
         assertThat(reporter.getTransactions()).isEmpty();
         client = getClient();
+        disableRecyclingValidation();
```

Comment:
[question] why do we need to do that for webflux (and same for vertx) ? As far as I know those were properly recycling (with the help from GC in the case of webflux).

Reply:
How spans are recycled with WebFlux and Vert.x is not affected by this change. What is affected are the tests that use the public API to get transactions, which then are not recycled (this is what this PR does 🙂 ).
But good point- we want to test recycling especially for WebFlux, so let me see how I can accommodate that (maybe switch to using the internal API instead of the public API)